PWGGA/GammaConv/AliAnalysisTaskNeutralMesonToPiPlPiMiNeutralMeson.cxx

```diff
@@ -110,6 +110,7 @@ AliAnalysisTaskNeutralMesonToPiPlPiMiNeutralMeson::AliAnalysisTaskNeutralMesonTo
   fPDGMassChargedPion(-1),
   fPDGCodeNDM(-1),
   fPDGCodeAnalyzedMeson(-1),
+  enableAsymmetryPlotCombCPionVsNPion(kFALSE),
```

Comment:
please follow naming convention putting f in front of global variables e.g. fEnableAsymmetry...